docs/rich_presence/Using_with_the_Game_SDK.mdx

```diff
@@ -21,11 +21,12 @@ Before we dig in, it's helpful to understand what Rich Presence data you can set
 | location                               | field name     | notes                                                                       |
 |----------------------------------------|----------------|-----------------------------------------------------------------------------|
 | First row below title                  | details        |                                                                             |
-| Second row below title                 | state          |                                                                             |
-| Second row below title                 | partySize      | In parenthesis next to the `state`, first number in the format `(1 of 10)`  |
-| Second row below title                 | partyMax       | In parenthesis next to the `state`, second number in the format `(1 of 10)` |
-| Third row below title                  | startTimestamp | Converted to a format such as `01:33 elapsed`                               |
-| First button at the bottom             | joinSecret     | Button has the text "Ask to join"                                           |
+| Second row below title                 | state          | If provided with no `partySize` or `partyMax`                               |
+| First badge in last row below title    | startTimestamp | Converted to a `mm:ss` format such as `12:01`                               |
+| Second badge in last row below title   | state          | If provided with a `partySize` and `partyMax`                               |
+| Second badge in last row below title   | partySize      | In parenthesis next to the `state`, first number in the format `(1 of 4)`   |
+| Second badge in last row below title   | partyMax       | In parenthesis next to the `state`, second number in the format `(1 of 4)`  |
+| First button at the bottom             | joinSecret     | Button has the text "Ask to Join"                                           |
 | Second button at the bottom            | spectateSecret | Button has the text "Spectate"                                              |
```

Comment:
the spectating feature was removed in 2020, we should just yeet that from the docs (incl the image)

Reply:
Oh! Very good to know. We were unsure if spectating was still supported, so defaulted to keeping it in the docs. But agreed we should rip it if the feature is no more.

Reply:
Ok removed all references to spectating!